Misc/NEWS.d/next/Library/2020-03-21-00-46-18.bpo-40017.HFpHZS.rst

```diff
@@ -0,0 +1 @@
+Added the CLOCK_TAI constant to the time library, if the operating system support is.
```

Comment:
Phrase it in the imperative. I.e., change "Added" to "Add".

(Now that I think of it "expose" might be better.)

remove the comma

`/is/it/`